static/compiler.js

```diff
@@ -504,13 +504,12 @@ define(function (require) {
     };
 
     Compiler.prototype.onCompilerSetDecorations = function (id, lineNums) {
-        if (id === this.id) {
+        if (id == this.id) {
```

Comment:
While testing this version, this check would not be successful even when it had to. Don't know why.